core/src/main/java/de/evoila/cf/broker/service/impl/BindingServiceImpl.java

```diff
@@ -74,15 +74,16 @@ public BaseServiceInstanceBindingResponse createServiceInstanceBinding(String bi
 		PlatformService platformService = platformRepository.getPlatformService(plan.getPlatform());
 
 		BaseServiceInstanceBindingResponse baseServiceInstanceBindingResponse;
+        String operationId = randomString.nextString();
 
-		if (async) {
-			if (platformService.isSyncPossibleOnBind()) {
-				baseServiceInstanceBindingResponse = syncCreateBinding(bindingId, serviceInstance,
+        if (async) {
+            if (platformService.isSyncPossibleOnBind()) {
+                baseServiceInstanceBindingResponse = syncCreateBinding(bindingId, serviceInstance,
 						serviceInstanceBindingRequest, plan);
-			} else {
-				bindingRepository.addInternalBinding(new ServiceInstanceBinding(bindingId, instanceId, null));
+                jobRepository.saveJobProgress(randomString.nextString(), bindingId, JobProgress.SUCCESS, "Synchronous binding.", operationId);
```

Comment:
Why only job progress creation for sync bindings in an async allowed environment? There is another case a few lines lower that also creates sync bindings.
Maybe a refactoring of the distinction of cases can remove the duplicate code segments. 

---
And please change the job description to a bit more descriptive. The action here can easily be confused with the name of the created object. I suggest something like: "Synchronous binding successfully created" or a simpler version: "Binding successfully created".